### ISSUE TITLE
Dashboard: Repetitive content on the page

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#19]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard) 
User Agent: Chrome, Firefox
Language: English

Issue Description: The section "Alerts and recalls in progress" seems to be repeated on the page. Its on top and then towards at the end as well. Both of these seem to have the same functionality with sorting/searching. Once there is a lot more information on the page, it can confuse the user. This can be confusing especially a screen reader user as there are 2 H2 headings that are same and have same table content.

Expected results: One of these should be removed.

![image](https://user-images.githubusercontent.com/14280062/127518176-2ff706ab-e40f-4bc6-aadb-c61940074c14.png)

**Comment by joel-osc** [comment]
Fixed.

**Comment by mmiron-HC** [comment]
@joel-osc  Please don't close the issues, just reassign to the tester. and put a note in the comments.

**Comment by qanitah** [comment]
This has been fixed
